apps/desktop/src/ui/MarkdownText.css

```
@@ -89,6 +96,9 @@
 }
 
 .markdown-text code {
+  max-inline-size: 100%;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)